cylc/flow/scheduler_cli.py

```diff
@@ -138,26 +138,30 @@ def parse_commandline(is_restart):
         parser = COP(RESTART_DOC, jset=True, argdoc=[SUITE_NAME_ARG_DOC])
     else:
         parser = COP(
-            RUN_DOC, jset=True,
+            RUN_DOC,
+            icp=True,
+            jset=True,
             argdoc=[SUITE_NAME_ARG_DOC, START_POINT_ARG_DOC])
 
     parser.add_option(
-        "--non-daemon", help="(deprecated: use --no-detach)",
-        action="store_true", default=False, dest="no_detach")
+        "-n", "--no-detach", "--non-daemon", help="Do not daemonize the suite",
+        action="store_true", dest="no_detach")
 
     parser.add_option(
-        "-n", "--no-detach", help="Do not daemonize the suite",
-        action="store_true", default=False, dest="no_detach")
+        "-a", "--no-auto-shutdown", help="Do not shut down"
```

Comment:
Could it confuse users? `-a` == `--no-auto-shutdown`... instead of `--auto-shutdown`...

Reply:
Yes, but I don't think I can change it due to issue with backward compatibility.

Reply:
Was afraid of that :cry: but all good here, :+1: +1 stands